### ISSUE TITLE
Official hyperparameters for ADFD dataset reproduction

### ISSUE THREAD

**Issue by as34r** [aikunyi/MedGNN#3]
I recently read your paper "Towards Multi-resolution Spatiotemporal Graph Learning for Medical Time Series Classification"with great interest and have been working on reproducing the results of the proposed method.
While the codebase is very helpful, I noticed that there is no provided bash script specifically for the ​​ADFD dataset​​. In order to reproduce the results, I tried running the code with parameter settings based on those mentioned in the paper. However, after several attempts, the performance I obtained still differs by around ​​5%​​ compared to the numbers reported in the paper.
Could you please kindly share the ​​recommended or official hyperparameter settings​​ (e.g., learning rate, batch size, number of epochs, model configurations, data preprocessing steps, etc.) that were used for the ​​ADFD dataset​​, so that I can better align my reproduction with the results presented in your work?
Thank you very much for your time and for sharing this insightful research. I really appreciate any guidance you can provide.

**Comment by aikunyi** [comment]
Thank you for your interest in our work. Regarding the parameters for **subject-based evaluation on the ADFD dataset**, you may find the following settings useful:

**d_model = 128
d_ff = 256
n_heads = 8
e_layers = 6
augmentations = “none”
resolution_list = “4,6,8”
nodedim = 8
batch_size = 64
learning_rate = 0.0002**


Please let me know if you need further clarification. I’d be happy to assist.

**Comment by as34r** [comment]
Thank you very much for your valuable suggestions during your busy schedule. I will try it out again later to see the results. Thank you.

**Comment by lzh2021-csu** [comment]
Dear authors,

Sorry to trouble you. I am also trying to reproduce the classification results of MedGNN. I noticed that in the previous discussion, you shared a recommended set of hyperparameters, so I also conducted my experiments based on those settings.

However, the results I obtained are still about 5% lower than the performance reported in the paper. Therefore, I would like to politely ask whether, in addition to these hyperparameters, there are any other key reproduction details that might have a noticeable impact on the final results.

Thank you very much for your time and for your help.

**Comment by as34r** [comment]
Jiahao&nbsp;Lian
***@***.***

**Comment by lzh2021-csu** [comment]
> Jiahao&nbsp;Lian
> ***@***.***

兄弟这是啥...

**Comment by as34r** [comment]
这是作者发我的参数信息，我也跑过发现没法完全达到论文所描述的那样，所以我就没顺下去看了






Jiahao&nbsp;Lian
***@***.***

**Comment by lzh2021-csu** [comment]
谢谢！请问你是只有这一个数据集无法复现嘛？我好几个数据集都跑不出论文的结果

**Comment by as34r** [comment]
我就复现了apava和adftd这两个





---Original---
From: ***@***.***&gt;
Date: Mon, Mar 30, 2026 20:01 PM
To: ***@***.***&gt;;
Cc: "Lian ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [aikunyi/MedGNN] Official hyperparameters for ADFD datasetreproduction (Issue #3)


lzh2021-csu left a comment (aikunyi/MedGNN#3)
 
谢谢！请问你是只有这一个数据集无法复现嘛？我好几个数据集都跑不出论文的结果
